tests/overrides/cors/envfile-filtered

```diff
@@ -3,4 +3,5 @@ NGINX_DEFAULT_SERVER_PORT=4321
 NGINX_CORS_ENABLED=1
 NGINX_CORS_DOMAINS=www\.example\.com
 NGINX_OSB_BUCKET=http://127.0.0.1:4321/bucket
+NGINX_OSB_PUBLIC_PATH=
```

Comment:
Are these `NGINX_OSB_PUBLIC_PATH` needed? They should be empty by default, as defined in the docker-entrypoint.

Reply:
tests will fail without it. We saw it yesterday

Reply:
That's correct. Sorry!